src/docs/clients/react-native/sourcemaps.mdx

```diff
@@ -40,6 +40,12 @@ node_modules/@sentry/cli/bin/sentry-cli releases \
     /path/to/your/files
 ```
 
+<Note>
+
+If you're using `sentry-cli` prior to version 1.59.0, pass `--rewrite` to the `upload-sourcemaps` command to fix up the source maps before the upload (inlines sources and so forth). Version 1.59.0 does this automatically.
```

Comment:
```suggestion
If you're using `sentry-cli` prior to version 1.59.0, pass `--rewrite` to the `upload-sourcemaps` command to fix up the source maps before the upload (inlines sources and so forth). Version 1.59.0 and newer does this automatically.
```
Would that work to make it "future proof"?

Reply:
The warning is a copy/paste from another docs page that has been reviewed before, but the change still makes sense, thanks.